code/modules/antagonists/disease/disease_mob.dm

```diff
@@ -42,7 +42,7 @@ the new instance inside the host to be updated to the template's stats.
 	var/move_delay = 1
 
 	var/next_adaptation_time = 0
-	var/adaptation_cooldown = 1200
+	var/adaptation_cooldown = 300
```

Comment:
```suggestion
	var/adaptation_cooldown = 600
```

I agree with prior feedback. 30 seems way too low.

Reply:
I say 90

Reply:
I would agree, but this isnt balanced for quick fucking bee rounds. the two minute evolution time can only evolve ten or so times if it spawns half way into our forty minute rounds. one minute, twenty or so evolutions. Note that if someone KNOWS how to play sentient disease or viro, they're going to also need to refund symptoms constantly to keep the ball rolling. As far as design goes. remember that sentient disease's ONLY influence on the game is their cough and their evolution panel, as well as their other two lackluster purchasable actives. it's a boring antag and very frustrating for people who do not know viro, which most people don't. it's good IF a virologist rolls it, but otherwise, it's the worst midround antag, honestly